sheerwater_benchmarking/forecasts/__init__.py

```diff
@@ -1,7 +1,7 @@
 """Forecasting models for the Sheerwater benchmarking project."""
 from .salient import salient
 from .abc import perpp
-from .ecmwf_er import ecmwf_abc_wb, ecmwf_ifs_er, ecmwf_ifs_er_debiased
```

Comment:
remove support for some of the abc functions in main (kept them in the abc frozen branch)